Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina
+
+# install git
+RUN apt-get -y update && apt-get install -y git
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN pip install git+https://github.com/openai/CLIP.git
+
+# for testing the image
+RUN pip install pytest && pytest -s -vv
+
+ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Comment:
```suggestion
FROM jinaai/jina:2.0.0rc6 as BASE

# install git
RUN apt-get -y update && apt-get install -y git

# setup the workspace
COPY . /workspace
WORKDIR /workspace

# install the third-party requirements
RUN pip install -r requirements.txt

# for testing the image
FROM BASE
RUN pip install -r tests/requirements.txt
RUN pytest -s -vv

FROM BASE
ENTRYPOINT ["jina", "pod", "--uses", "config.yml"]
```

Reply:
We should avoid downloading the model every time the user starts the executors.

Reply:
BASE should be lowercased as repository name must be lowercase ?

>